client/dashboard/domains/domain-overview/icann-suspension-notice.tsx

```diff
@@ -5,30 +5,47 @@ import {
 	__experimentalText as Text,
 	__experimentalVStack as VStack,
 } from '@wordpress/components';
-import { useDispatch } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
-import { store as noticesStore } from '@wordpress/notices';
+import { useAnalytics } from '../../app/analytics';
 import Notice from '../../components/notice';
 
 export default function IcannSuspensionNotice( { domainName }: { domainName: string } ) {
-	const resendIcannVerificationEmail = useMutation(
-		resendIcannVerificationEmailMutation( domainName )
-	);
-	const { createSuccessNotice, createErrorNotice } = useDispatch( noticesStore );
+	const resendIcannVerificationEmail = useMutation( {
+		...resendIcannVerificationEmailMutation( domainName ),
+		meta: {
+			snackbar: {
+				success: __(
+					'Verification email sent! It will arrive in a few minutes. Check your inbox and follow the instructions to verify your domain name.'
```

Comment:
Nice idea updating the copy! It's a nitpick, but what do you think of the following slight adjustment?


```suggestion
					'Verification email sent! It should arrive within a few minutes. Please check your inbox and follow the instructions to verify your domain name.'
```